src/java.xml/share/classes/com/sun/org/apache/xalan/internal/xsltc/trax/TransformerImpl.java

```diff
@@ -48,6 +48,8 @@
 import java.net.URL;
 import java.net.URLConnection;
 import java.net.UnknownServiceException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
```

Comment:
Nit: you should not need to use Paths. `Paths.get(URI)` just calls `Path.of(URI)`

Reply:
Right, but Path.of was introduced in JDK 11. I hope to avoid more advanced features if we can keep the code level at 8. There had been previous cases where we stayed at 8. So far, only a few classes need to be modified for java.xml to compile at 8.